base/scene.py

```diff
@@ -22,5 +23,5 @@ def render(self):
     def handle_input(self):
         pass
 
-    def transition_to(self, scene_name):
-        self.scene_manager.transition_to(scene_name)
+    def transition_to(self, scene_name, **kwargs):
```

Comment:
Yea I would prefer that, maybe scene_context or something that better defines what it is.